### DISCUSSION TITLE
sh create-remix command not found

### DISCUSSION THREAD

**Discussion by Colinwood11** [remix-run/remix#3212] [Q&A]
when i running create, but

npx create-remix
Need to install the following packages:
  create-remix
Ok to proceed? (y) y
npm WARN deprecated source-map-url@0.4.1: See https://github.com/lydell/source-map-url#deprecated
npm WARN deprecated urix@0.1.0: Please see https://github.com/lydell/urix#deprecated
npm WARN deprecated resolve-url@0.2.1: https://github.com/lydell/resolve-url#deprecated
npm WARN deprecated rollup-plugin-inject@3.0.2: This package has been deprecated and is no longer maintained. Please use @rollup/plugin-inject.
npm WARN deprecated source-map-resolve@0.5.3: See https://github.com/lydell/source-map-resolve#deprecated
sh: create-remix: command not found

Now using node v18.1.0 (npm v8.8.0)

**Comment by deflexable** [comment]
an also having the issue, @Colinwood11 did you fix it

**Reply by Colinwood11** [reply]
Since I was just creating it out of curiosity, I didn't check it in detail. Sorry I can't help you
